actions/src/main/kotlin/com/alfresco/content/actions/ActionUploadFiles.kt

```diff
@@ -0,0 +1,45 @@
+package com.alfresco.content.actions
+
+import android.content.Context
+import android.view.View
+import com.alfresco.content.ContentPickerFragment
+import com.alfresco.content.data.Entry
+import com.alfresco.content.data.OfflineRepository
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+/**
+ * Marked as ActionUploadFiles for uploading the files
+ */
+data class ActionUploadFiles(
+    override var entry: Entry,
+    override val icon: Int = R.drawable.ic_action_upload,
+    override val title: Int = R.string.action_upload_files_title
+) : Action {
+
+    private val repository = OfflineRepository()
+
+    override suspend fun execute(context: Context): Entry {
+        val result = ContentPickerFragment.pickItems(context, MIME_TYPES)
+        if (result.count() > 0) {
```

Comment:
Can we use isEmpty here?

Reply:
we could use empty here but in existing code, count is being checked so following same.

Reply:
we could use empty here but in existing code, count is being checked so following same.